.buildkite/scripts/steps/console_definitions_sync.sh

```diff
@@ -3,70 +3,111 @@ set -euo pipefail
 
 GIT_SCOPE="src/platform/plugins/shared/console/server/lib/spec_definitions"
 
+# We will generate the console definitions for both main and 8.x branches
+BRANCHES=("main" "8.x")
```

Comment:
Did we consider configuring the pipeline to run both branches instead? There will be less branch changing and the updates can run in parallel.  As this currently is, checking out 8.x will be out of date by up to a day.

https://github.com/elastic/kibana/blob/d7513a1ed1eb8fb43fb305e9fbd085cb3d814945/.buildkite/pipeline-resource-definitions/kibana-console-definitions-sync.yml#L25 + another cron.  Branch can be referenced by `$BUILDKITE_BRANCH`, e.g. `git clone --branch "$BUILDKITE_BRANCH" https://github.com/elastic/elasticsearch-specification --depth 1;`

Reply:
To be honest I didnt consider it as I didnt know it could be a thing. If I understand correctly what you are saying, we could more or less leave the script as is (but using the $BUILDKITE_BRANCH where needed) and create a separate `kibana-console-definitions-sync.yml` file that is more or less the same as the other one but instead has a different `branch_configuration` (one for main and one for 8.x)? 🤔

Reply:
Yep that's correct

Reply:
Dope, I've made the changes let me know what you think